dbt-cta/facebook_marketing/models/1_cta_incremental/_1_cta_incremental__models.yml

```diff
@@ -20,13 +20,14 @@ models:
         description: ''
         tests:
           - not_null
-          # - relationships:
-          #     to: ref('ads_base')
-          #     field: id
-          #     config:
-          #       severity: error
-          #       error_if: ">10"
-          #       warn_if: ">0"
+          - relationships:
+              to: ref('ads_base')
+              field: id
+              config:
+                severity: error
+                error_if: ">10"
+                warn_if: ">0"
```

Comment:
ooooh love this error_if vs warn_if situation

Reply:
commented!